content/code-of-conduct-reporting-harrassment.md

```diff
@@ -1,5 +1,6 @@
 type: page
 url: code-of-conduct/reporting-harrassment/index.html
+title: Reporting harrassment
 ---
 
 ## How to report harrassment
```

Comment:
harrassment -> harassment :smile: 

Might be worth a search and replace for this.